talent-digital/talentkit
# Feature/ti 2758 integrate talentkit with badges endpoint

This PR changes the internal storage of awarded badges. Previously we used savegame mechanism now we have a dedicated endpoint for that.

To test:
- Go to canary app
- There should be unearned 1 badge (opacity set to 0.5)
- Click on the Award button (check if a BE request is made)
- Click Rerender button or refresh page - badge should now be awarded
